background.ts

```diff
@@ -219,7 +220,12 @@ async function doFirstRun() {
   if (!settings.isFirstRun()) {
     return;
   }
-  await window.openFirstRun();
+  try {
+    inFirstRun = true;
```

Comment:
And this line can be removed:
```suggestion
```

Reply:
I renamed a method to clarify the two concepts.

Reply:
Yes, but it doesn't address the race condition: `inFirstRun` is still `false` until `doFirstRun()` is called. When you get the "Activate" request before you have a chance to start `doFirstRun` you will still raise the main window.

Reply:
That's a good point.